t/fixtures/migrate.sh

```diff
@@ -203,6 +203,32 @@ setup_multiple_local_branches() {
   git checkout master
 }
 
+# setup_multiple_local_branches_with_alternate_names performs the same task
+# as setup_multiple_local_branches, but creates a file with no extension.
+setup_multiple_local_branches_with_alternate_names() {
+  set -e
+
+  reponame="migrate-info-multiple-local-branches"
+
+  remove_and_create_local_repo "$reponame"
+
+  base64 < /dev/urandom | head -c 120 > no_extension
+  base64 < /dev/urandom | head -c 140 > a.txt
```

Comment:
I'll just make a note here that `head -c` isn't portable across systems. OpenBSD, for example, lacks it, and POSIX doesn't require it.

I don't think that's critical in this case, since we're basically doing the same thing slightly above, so anyone for whom this is a problem won't be significantly worse off than before.